game/ui/inventorymenu.cpp

```diff
@@ -105,7 +105,7 @@ void InventoryMenu::close() {
   }
 
 void InventoryMenu::open(Npc &pl) {
-  if(pl.isDown() || pl.isMonster() || pl.isInAir() || pl.isSlide() || (pl.interactive()!=nullptr))
+  if(pl.isDown() || pl.isMonster() || pl.isInAir() || pl.isSlide() || pl.interactive()!=nullptr)
```

Comment:
nitpick: code-style only change - not related to PR

Reply:
So this should be a separate PR?

Reply:
No - I'll fix it as separated commit, as followup